### PR TITLE
Changed dir structure, some variable names, and added pickup the data…

### PR DESCRIPTION
**Pull request by anabellaspinelli**
… from the create animal call.

### CODE REVIEW

File: src/services/animals.js [hunk 1]
```diff
@@ -19,19 +21,19 @@ class Animals {
     return animal;
   }
 
-
   async create(data, params) {
-    const Model = require('../models/animal');
```
Comment (anabellaspinelli) [inline src/services/animals.js:24]: Moved to global so we don't require it on each `create` call

File: src/models/animals.js [hunk 2]
```diff
@@ -16,19 +16,20 @@ const AnimalSchema = new Schema({
     type: String,
     required: true
   },
-  birthday: {
+  dateOfBirth: {
```
Comment (anabellaspinelli) [inline src/models/animals.js:19]: Changed to reflect the actual date of birth (from which we can extract the birthday if we want to)

File: src/models/animals.js [hunk 3]
```diff
@@ -16,19 +16,20 @@ const AnimalSchema = new Schema({
...
     type: Date,
     required: true
   },
   description: {
     type: String,
     required: true
   },
-  ingressDate: {
+  entryDate: {
```
Comment (anabellaspinelli) [inline src/models/animals.js:27]: Though "ingress" is correct, I think "entry" is more clear. WDYT @smarbos?

File: src/models/contracts.js [hunk 4]
```diff
@@ -2,10 +2,12 @@
...
 const mongoose = require('mongoose');
 
-const Schema = mongoose.Schema;
+const Schema = mongoose.Schema,
+      ObjectId = Schema.ObjectId;
+
 const ContractsSchema = new Schema({
   _id: {
-    type: ObjectId,
+    type: Number,
```
Comment (anabellaspinelli) [inline src/models/contracts.js:10]: This was giving me "ObjectId" is not defined (I added it above), but also: 

https://stackoverflow.com/questions/8111846/how-to-set-objectid-as-a-data-type-in-mongoose

File: src/models/contracts.js [hunk 5]
```diff
@@ -16,7 +18,7 @@ const ContractsSchema = new Schema({
     type: ObjectId,
     required: true
   },
-  timeStamp: {
+  createdAt: {
```
Comment (anabellaspinelli) [inline src/models/contracts.js:21]: Just to clarify what the timestamp represents, and not to expose implementation.

File: src/services/animals.js [hunk 6]
```diff
@@ -19,19 +21,19 @@ class Animals {
...
+      stage: data.stage,
+      name: data.name,
+      dateOfBirth: data.dateOfBirth,
+      description: data.description,
+      entryDate: data.entryDate
     };
-    const animal = new Model(Animaldata);
-    this.animals.push(animal);
+
+    const animal = new AnimalModel(AnimalData);
```
Comment (anabellaspinelli) [inline src/services/animals.js:33]: If you get `Cannot POST /animals` error (but then see it persists anyway) I think the error happens in this line. I should investigate mongoose more to understand what's wrong.

File: src/models/contracts.js (hunk 4 above)
Comment (smarbos) [reply src/models/contracts.js:10]: I double checked and the _id is not necessary. Provide it is not a problem, but if we dont specify it, a field _id with type ObjectId get created in the new document saved in the DB.

**Review by smarbos — APPROVED** [review]
Great! Thanks =D

**Pull request merged**